src/index.ts

```diff
@@ -8,6 +8,8 @@ import Start from './commands/command.start';
 
 const program = new Command();
 
+program.version('0.0.5');
```

Comment:
Please explain purpose of this?

Reply:
This allows user to check the version of oi installed on the system.. oi -V or oi --version